test/test_events.py

```diff
@@ -1,32 +1,27 @@
+from __future__ import absolute_import
```

Comment:
The changes in this module are either small tweaks to accommodate the new implementation (e.g. expect more connections now) or cleanup where we stop using the `rabbit_manager` to inspect queues (slow and flakey) in favour of the new `queue_info` mechanism that determines queue info using AMQP.